pkg/terra/client/gas_price_estimator.go

```diff
@@ -0,0 +1,128 @@
+package client
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"reflect"
+	"strings"
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+type GasPriceEstimator interface {
+	MustGasPrice(denom []string) []sdk.DecCoin
+}
+
+var _ GasPriceEstimator = (*FixedGasPriceEstimator)(nil)
+
+type FixedGasPriceEstimator struct {
+	gasPrice sdk.DecCoin
+}
+
+func NewFixedGasPriceEstimator(price sdk.DecCoin) *FixedGasPriceEstimator {
```

Comment:
The terra txm could potentially fallback to a fixed gas price estimator based on the fallbackGasPrice we currently have in chains config if theres an issue with NewFCDGasPriceEstimator, but I kind of like the fail fast on boot if the fcd url is problematic (and just removing the fallbackGasPrice). Wdyt @jmank88 ?

Reply:
I think that would work fine as long as we only fail to start the _terra.Chain_ and don't block the node starting up as a whole.

Reply:
yeah that should be doable

Reply:
found a better way